readme.md

```diff
@@ -17,5 +17,5 @@ You can find some sample screenshots in the [backstop_data/bitmaps_reference](ba
 
 **Result**
 
-[Demo link](https://mate-academy.github.io/layout_task-boilerplate-ci/) <br>
-[Test report](https://mate-academy.github.io/layout_task-boilerplate-ci/report/html_report/)
+[Demo link](https://ilovepinkpony1.github.io/fs_apr19_vitalii_o/) <br>
+[Test report](https://ilovepinkpony1.github.io/fs_apr19_vitalii_o/report/html_report/)
```

Comment:
Add your repo name `layout_colored-blocks` instead `fs_apr19....`. Here should be your repo name, not a branch.